src/api/app/controllers/webui/cloud/azure/configurations_controller.rb

```diff
@@ -27,7 +32,15 @@ def destroy
         private
 
         def set_azure_configuration
-          @azure_configuration = User.session!.azure_configuration || User.session!.create_azure_configuration
+          @azure_configuration = User.session ? find_or_create_azure_notification : ::Cloud::Azure::Configuration.new
```

Comment:
This is a chicken-egg problem: we need `User` to fetch his `@azure_configuration` but we need the `@azure_configuration` in the `Policy`.